com.liferay.blade.eclipse.provider/src/com/liferay/blade/eclipse/provider/SSEXMLFile.java

```diff
@@ -82,5 +83,61 @@
 		return _results;
 	}
 
+	@Override
+	public Collection<SearchResult> findLegacyDependency(String groupId, String artifactId) {
```

Comment:
what if I just add a 
`Collection<SearchResult> findLegacyDependency(String parentNodeName, Map<String,String> childrenNodeNameValueMap);`
in XMLFile.java and parentNodeName is for parent Node name and map is it's child node name and value map , only some parent node has all the children node and all the value is matched , then checker will return a SearchResult. This is a little common method but now it is just for finding dependency in pom.